README.md

```diff
@@ -31,7 +31,9 @@ this is probably your `AppDelegate`'s `...didFinishLaunchingWithOptions:` method
 import SplunkOtel
 ...
 // Your beaconUrl and rumAuth will be provided by your friendly Splunk representative
-SplunkRum.initialize(beaconUrl: "https://rum-ingest.us0.signalfx.com/v1/rum", rumAuth: "ABCD...")
+SplunkRumBuilder(beaconUrl: "https://rum-ingest.us0.signalfx.com/v1/rum", rumAuth: "ABCD...")
+  // Call functions to configure additional options
+  .build()
```

Comment:
How do you add the environment here? Would this be accurate? We should add this to the official docs:

```swift
import SplunkOtel
//..
SplunkRumBuilder(beaconUrl: "https://rum-ingest.<realm>.signalfx.com/v1/rum", rumAuth: "<rum-token>")
// Call functions to configure additional options
   .globalAttributes(globalAttributes: ["deployment.environment": "<environment>"])
    .build()
```

Reply:
```swift
SplunkRumBuilder(beaconUrl: "https://rum-ingest.<realm>.signalfx.com/v1/rum", rumAuth: "<rum-token>")
   .globalAttributes(globalAttributes: [:])
  .deploymentEnvironment(environment: "<environment>")
   .build()
```